keps/prod-readiness/sig-network/4762.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 4762
+alpha:
+  approver: "@jpbetz"
```

Comment:
Joe is OOO this week - can @deads2k or @johnbelamaric or @soltysh take this one?

Reply:
Switch to my handle.